easybuild/easyconfigs/p/pyEGA3/pyega3-4.0.0-fixRequirements.patch

```diff
@@ -0,0 +1,31 @@
+pyEGA3 has strict version requirements, but included tests run even when
+using newer versions.
+Author: Orient Kružík (INUITS)
+diff -ruN pyega3-4.0.0_orig/pyega3.egg-info/requires.txt pyega3-4.0.0/pyega3.egg-info/requires.txt
+--- pyega3-4.0.0_orig/pyega3.egg-info/requires.txt	2021-12-09 14:09:53.000000000 +0100
++++ pyega3-4.0.0/pyega3.egg-info/requires.txt	2022-03-18 16:17:16.080009959 +0100
+@@ -1,5 +1,5 @@
+-requests==2.26.0
+-tqdm==4.19.6
+-htsget==0.2.5
+-psutil==5.6.6
+-urllib3==1.26.7
++requests>=2.26.0
++tqdm>=4.19.6
++htsget>=0.2.5
++psutil>=5.6.6
++urllib3>=1.26.7
```

Comment:
Why don't we avoid using a patch file for this, and just use a `sed` command in `preinstallopts`?

```python
'preinstallopts': "sed -i 's/==/>=/g' setup.py && ",